_gsocproposals/2021/proposal_TMVADLConv3DGPU.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TMVA Deep Learning Developments - 3D Convolutions for GPU
+layout: gsoc_proposal
+project: TMVA
+year: 2020
```

Comment:
Year should be 2021

Reply:
Thanks for spotting this. It is updated now !